Unwrap/Reusables/CountingLabel.swift

```diff
@@ -47,24 +47,22 @@ class CountingLabel: UILabel {
 
     /// Called every time a frame draw happens thanks to CADisplayLink, so its our chance to adjust our current value towards our target value, then update our label.
     @objc func updateCount() {
+        
+        // Target reached; clamp it and stop doing more work.
+        let stopWorking = {
+            self.currentValue = self.targetValue
+            self.displayLink?.invalidate()
+        }
+        
         if targetValue > fromValue {
             // We need to add numbers to reach our target.
             currentValue += changeSpeed
-
-            if currentValue > targetValue {
-                // Target reached; clamp it and stop doing more work.
-                currentValue = targetValue
-                displayLink?.invalidate()
-            }
+            if currentValue > targetValue { stopWorking() }
```

Comment:
This should be written across more lines: an empty line before the condition starts, then `stopWorking()` on a line of its own, and the closing brace on a line of its own. Unless I missed something, the only time Unwrap code puts conditions on one line only when using `return` as part of a `guard` early return.

Reply:
Awesome. If you could make the closure into a function and fix the new lines, then push up your changes, I can merge.